client/scripts/views/pages/discussions/index.ts

```diff
@@ -63,23 +63,42 @@ const getLastSeenDivider = (hasText = true) => {
   );
 };
 
+
 export const CommunityOptionsPopover: m.Component<
   { isAdmin: boolean; isMod: boolean },
   {}
 > = {
   view: (vnode) => {
     const { isAdmin, isMod } = vnode.attrs;
     if (!isAdmin && !isMod && !app.community?.meta.invitesEnabled) return;
+
+    // add extra width to compensate for an icon that isn't centered inside its boundaries
+    const DropdownIcon = m('div', {
+      style: {
+        display: 'flex',
+        flexDirection: 'row',
+        alignItems: 'center',
+        cursor: 'pointer',
```

Comment:
We try to avoid inline styling in almost all cases across the site. The only time we don't is with some tricky table and table cell formatting stuff. This and the css below should be moved into an aptly named scss file and the styles imported appropriately.